tsconfig.json

```diff
@@ -11,8 +11,9 @@
     "jsx": "react",
     "rootDir": "src",
     "outDir": "lib/components",
-    "incremental": true
+    "incremental": true,
+    "allowJs": true
```

Comment:
With allowJs set to true, we transpile the generated bundle down to ES2015. 
I see one problem and one potential risk with this approach:
- Risk: we modify the bundled dependencies by transforming them to a different target - not 100% sure how that's handled from a licence perspective (todo for me: read through and do research about).
- Problem: `d3-scale`'s ` `Internmap` dependency extends the native Map object ([link to source](https://github.com/mbostock/internmap/blob/main/src/index.js#L1)). When compiling to es15, it doesn't emit class constructs because ES5 didn't have them. You can see this issue in action when checking out this PR, run `npm run start` and navigate to the http://localhost:8080/#/light/bar-chart/basic example.

A solution would be to add a Map polyfill (feels not right) or to manually copy the bundle from src/internal/vendor to lib/components/internal/vendor.

With the latest option we'd be back again at the initial PR #399.

I suggest to move on with the initial PR and attach this issue to the "Target ES2019" PR (#144). As soon as we can move on with changing the target this would not be a problem anymore.

Reply:
Alright, let's keep that as an improvement idea for BRF.

We can probably bundle the original version w/o transpiling. The allow-js was for our main build to include the vendor deps to the build output.